### PR TITLE
Add setup script with explicit paths to installation locations

### PR DESCRIPTION
**Pull request by mindyls**
Include a simple script that is intended to be used as reference / modified by individual users, based on their specific installation locations.  This script does not expect any relationships between installation directories (ie, conda installation path, test data repos, source repos, cartopy shapefiles, etc), but is intended as a guide for users to configure their own environment.

```
$ source $HOME/geoips/geoips/setup/USER_MODIFY_config
(geoips) $ which python
$HOME/mambaforge/envs/geoips/bin/python
(geoips) $ $GEOIPS_PACKAGES_DIR/geoips/tests/integration_tests/full_test.sh
```

### CODE REVIEW

File: docs/source/releases/v1_11_3a0.rst [hunk 1]
```diff
@@ -15,6 +15,7 @@ Version 1.11.3a0 (2023-08-19)
 
 * Require sphinx<7.2 due to bugs introduced 2023-08-17 7.2.x release
 * Add .vscode settings repository to "full_install.sh"
+* Add script to enable geoips environment
```
Comment (mindyls) [inline docs/source/releases/v1_11_3a0.rst:18]: NOTE: Do not merge until other PRs are merged, and this one is automatically updated to point to main!

File: setup/USER_MODIFY_config [hunk 2]
```diff
@@ -0,0 +1,27 @@
+
+# REQUIRED Set this path to the parent directory where you cloned the geoips repo.
+# ie, $GEOIPS_PACKAGES_DIR/geoips/tests/integration_tests/base_test.sh
+export GEOIPS_PACKAGES_DIR=$HOME/geoips
+
+# OPTIONAL Set this path to where conda is installed
+# This is not required if you activated conda in your .bashrc
+eval "$($HOME/mambaforge/bin/conda shell.bash hook)"
```
Comment (cpcamacho) [inline setup/USER_MODIFY_config:8]: I guess "eval" is needed here to work? Probably not ideal, but likely fine?

**Review by cpcamacho — APPROVED** [review]

File: setup/USER_MODIFY_config (hunk 2 above)
Comment (jsolbrig) [reply setup/USER_MODIFY_config:8]: I wonder if that line should be commented out by default?

File: docs/source/releases/v1_11_3a0.rst (hunk 1 above)
Comment (jsolbrig) [reply docs/source/releases/v1_11_3a0.rst:18]: Done and ready to go.

File: setup/USER_MODIFY_config (hunk 2 above)
Comment (mindyls) [reply setup/USER_MODIFY_config:8]: It is the line that conda tells you to run if you tell it not to initialize the shell

Comment (mindyls) [reply setup/USER_MODIFY_config:8]: Yeah,  sure,  comment it out by default

Comment (jsolbrig) [reply setup/USER_MODIFY_config:8]: I think I'm going to approve this as a "hidden Mindy-only feature" for now. She uses this for her environment initialization. I've got my own methods, too.

**Review by jsolbrig — APPROVED** [review]

**Pull request merged**